devtools/conda-recipe/meta.yaml

```diff
@@ -48,6 +48,6 @@ test:
    
 about:
   home: https://github.com/MobleyLab/Lomap
-  license: LGPL
+  license: MIT
   license_file: LICENCE.txt
   summary: Alchemical free-energy calculations planner
```

Comment:
could be worth referencing the original paper url as well, e.g. https://link.springer.com/article/10.1007%2Fs10822-013-9678-y